index.html

```diff
@@ -22,11 +22,17 @@
       <h4 class="logo">emmanuel moombe</h4>
       <div class="humber">
          <i class="fa-solid fa-bars"></i>
+        <i class="fa fa-fw fa-x close-btn"></i>
+         <ul class="menu-mobile">
+          <li class="menu-item"><a href="#work">portfolio</a></li>
+          <li class="menu-item"><a href="#aboutSection">About</a></li>
+          <li class="menu-item"><a href="#formSection">contact</a></li>
+        </ul>
```

Comment:
Your menu works well, but its styling is vastly different than the one featured on the Figma template.

Therefore:

- [ ] Please, make sure the menu is exactly styled as shown in the Figma design. 

## Hints

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/162736581-328927e0-ede1-46d0-b655-766a39c4eee6.png) | ![image](https://user-images.githubusercontent.com/57421823/162736616-f593c1e5-d80a-4a5e-8c40-fb5c2d949cd5.png)

General hints:

- Align the items more to the left, exactly as the Figma.
- Move the items more to the top, exactly as the Figma.
- Make sure the horizontal lines are the exact color and the exact length.
- Change "portfolio" for "Portfolio".
- Change "contact" for "Contact".
- The font stylings do not match.  This includes font-size, font-weight, font-family, line-height, color.
  - This is how to get the exact font styling:

![image](https://user-images.githubusercontent.com/57421823/162737660-718be085-9083-48c6-bea5-d6c15dd54a12.png)

- Check your margins and padding.  Make sure they match.  

If you implement this, your menu will look like the Figma! 💪

Reply:
All good now 👍🏽